### ISSUE TITLE
Adding Universal App support

### ISSUE THREAD

**Issue by PedroLamas** [MvvmCross/MvvmCross#656]
Looking as how MvvmCross is currently being structured, I'm not entirely sure that one can add Universal Apps support just by fiddling with the Profile in the PCL's...

Let's take `CrossCore` as a starting point, I see that the main CrossCore PCL library uses `Framework4.0, Profile158`, giving the following output:

- net4.5
- sl5
- win8
- wp8
- android
- ios

There is no way I can select WPA8.1 or WP8.1 with this, it will always say that it is an incompatible configuration...

I've taken a look at how Json.net is doing it, and apparently they use a Portable for `Framework4.0, Profile136` and another for `Framework4.5, Profile225`.

The 1st one includes Xamarin Android and iOS support, while the second one works with Universal Apps.

Not sure were he got `Profile225` (doesn't show the platforms in my VS2013), but the information I got from Microsoft is to use `Profile259` (portable-net45+wp8+win8+wpa81)

Bottom line, I'm tempted in actually duplicating CrossCore with this approach, one to `Framework4.0, Profile136` and one for `Framework4.5, Profile259`.

Thoughts?

**Comment by martijn00** [comment]
With all the changes around the nuspecs, uwp apps, and vs2015 i think we can close this one. If there are still issues left here please open new separate issues which can be fixed in smaller parts.
